### ISSUE TITLE
Nav is broken in Learn/Performance article

### ISSUE THREAD

**Issue by wbamberg** [mdn/content#29240]
See https://github.com/mdn/content/issues/29226#issuecomment-1729892916.

We should at least fix the Next links, and probably repurpose https://developer.mozilla.org/en-US/docs/Learn/Performance/Web_Performance_Basics so it makes more sense in the context of the Learn/Performance module.

**Comment by caugner** [comment]
@wbamberg Are you planning to work on this?

- If yes, would you mind assigning the issue to yourself?
- If not, could you please use the issue template to allow someone else to pick it up?

The way I see it:

1. That page is missing in the sidebar (needs a yari PR).
2. The `PreviousMenu` macro is missing at the bottom (needs a content PR).

**Comment by wbamberg** [comment]
There's a longer analysis of this issue at https://github.com/mdn/content/issues/29226#issuecomment-1730012918.

Just adding the formal nav elements (e.g. Next macro call) doesn't resolve the incoherence of having a page like this in a place like that, and fixing that takes deeper content changes, so it actually makes sense in context.

> Are you planning to work on this?

I probably will eventually unless someone else wants to first.

> If not, could you please use the issue template to allow someone else to pick it up?

I'm a bit unclear what currently is stopping someone else from working on this issue.

**Comment by wbamberg** [comment]
> The way I see it:
> 
>     1. That page is missing in the sidebar (needs a yari PR).
> 
>     2. The `PreviousMenu` macro is missing at the bottom (needs a content PR).

I don't think this alone would be a good fix. The underlying issue is that https://developer.mozilla.org/en-US/docs/Learn/Performance/Web_Performance_Basics doesn't make sense as the final page in the Performance learn module, seeing as it is a slightly random collection of performance tips that doesn't acknowledge its existence as part of a wider series of articles, probably because it wasn't part of a series of articles when it was written.

So "Nav is broken in Learn/Performance article" is probably not a very good title for this, and maybe ""Decide what to do about 'Performance basics'" would be better. Maybe we should delete it, or keep it, rewrite it, and fix the nav.

> Are you planning to work on this?

Not right now, but maybe eventually if noone else does. I think a proper fix is a lot of work though.

**Comment by Rumyra** [comment]
This sounds to me like more of a project suggestion than an issue - am I correct @wbamberg ? If so I can label accordingly and transfer into the roadmap backlog

**Comment by wbamberg** [comment]
> This sounds to me like more of a project suggestion than an issue

What are the criteria for deciding that, and would it still be visible in this issue list? "transfer" makes it sound like it wouldn't.

**Comment by Rumyra** [comment]
There's guidelines here https://github.com/orgs/mdn/projects/26/views/7?pane=info - but re-reading it, it seems like we want to make this more of a guide rather than part of the learning area, and put it back with /Web/Performance (?)
